user/mixins/models.py

```diff
@@ -111,4 +112,21 @@ def sections_complete_for(self, text: Text) -> int:
                                     .first() \
                                     .number_of_sections
 
-        return sections_complete
\ No newline at end of file
+        return sections_complete
+
+    def vote_history(self, text: Text) -> str:
+        try:
+            v = TextRating.objects \
+                          .filter(text_id=text.id) \
+                          .first() \
+                          .vote
+            if v == 1:
+                return "up"
+            elif v == 0:
+                raise ValueError
+            else:
```

Comment:
Might be better to explicitly check for `-1` for down, and make all other values a `ValueError`

Reply:
resolved in commit 89d1157